js/remove_anchor_from_url.js

```diff
@@ -5,4 +5,9 @@ function removeUrlAnchor(url) {
     return urlSplitted[0];
 }
 
-removeUrlAnchor('www.codewars.com#about')
\ No newline at end of file
+function removeAfterHash(s){
+      return s.replace(/\#.*/,'');
```

Comment:
Quite clever 😄 Personally I would use `/(.*)#/` and use the first capturing group, but this seems simpler. Nice work